### PR TITLE
[FIX] forwardport: clarify forward port message

### PR DESCRIPTION
**Pull request by mart-e**
The message was not clear
- don't know if anything is needed from the author
- don't know if the last step in the chain

Remove ping message and include it in the first one,

Maybe a documentation page would clarify it
_edit: done https://github.com/odoo/odoo/wiki/Mergebot#forward-port_

### CODE REVIEW

**Comment by xmo-odoo** [comment]
The issue I have with this is the PR message becomes a commit in case of merge or rebase-merge, so the more it strays from the original the more garbage we will end up with.

File: forwardport/models/project.py [hunk 1]
```diff
@@ -433,11 +415,58 @@ def _port_forward(self):
...
+
+To merge the full chain, say
+> @fw-bot r+
+
+More info at
+https://github.com/odoo/odoo/wiki/Mergebot#forward-port
+""" % (target.name, chain_msg)
+            else:
+                message = """
+This PR targets %s and is part of the forard-port chain.
```
Comment (KangOl) [inline forwardport/models/project.py]:
```suggestion
This PR targets %s and is part of the forward-port chain.
```

File: forwardport/models/project.py [hunk 2]
```diff
@@ -433,11 +415,58 @@ def _port_forward(self):
...
+Either perform the forward-port manually (and push to this branch, proceeding
+as usual) or close this PR (maybe?).
+
+In the former case, you may want to edit this PR message as well.
+""" % (h, source.number, out, err)
+            elif base.limit_id == target:
+                chain, current = [pr.number], pr
+                while current.parent_id and current.parent_id != source:
+                    chain.append(current.parent_id)
+                    current = current.parent_id
```
Comment (xmo-odoo) [inline forwardport/models/project.py]: this could probably use _iter_ancestors

File: forwardport/models/project.py [hunk 3]
```diff
@@ -433,11 +415,58 @@ def _port_forward(self):
...
+                chain_msg = "\n* ".join("#%s" % c.number for c in chain)
+                message = ping + """
+This PR targets %s and is the last of the forward-port chain containing:
+* %a
```
Comment (xmo-odoo) [inline forwardport/models/project.py]: why %a?

Comment (mart-e) [reply forwardport/models/project.py]: because typo

**Comment by mart-e** [comment]
Fixed but obviously not tested

**Comment by xmo-odoo** [comment]
> Fixed but obviously not tested

400px-GLaDOS_P2

**Pull request closed without merging**